src/vm/jitinterface.cpp

```diff
@@ -10914,27 +10939,7 @@ void CEEJitInfo::reserveUnwindInfo(BOOL isFunclet, BOOL isColdCode, ULONG unwind
 
```

Comment:
I am not sure if I understand. The UnwindData is set by the runtime (in CEEJitInfo::allocUnwindInfo) and extracted by the runtime (in RtlpGetFunctionEndAddress), so there is no need for contract between those two.